src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -80,12 +78,6 @@ const DefaultTranscriptslist = (props: Props) => {
 
   const { gene } = props;
 
-  //Using this to get the default order of transcripts in which the first one is selected, this might change later with the data coming directly from thoas
```

Comment:
i think this code need to stay because we need to apply the default sorting first and then display the transcripts. sorry if i confused you when we spoke but we do need this default sorting and when you apply the default sorting, the first transcript will be either "Mane" or "Canonical" or can be empty.

Reply:
Agreed we dont need this piece of code but will be good to use `useMemo` to apply sorting when the geneId or sortingRule are different